php/WP_CLI/Dispatcher/CommandNamespace.php

```diff
@@ -28,7 +28,8 @@ public function show_usage() {
 		$count = 0;
 
 		foreach ( $methods as $name => $subcommand ) {
-			$prefix = ( 0 == $i++ ) ? 'usage: ' : '   or: ';
+			$prefix = ( 0 === $i ) ? 'usage: ' : '   or: ';
+			++$i;
```

Comment:
As there is no other logic to this statement apart from the increment, we should default to the more common postfix increment, like we do in other parts of the code.
```suggestion
			$i++;
```